docs/content/operations/druid-console.md

```diff
@@ -45,49 +45,69 @@ The home view provides a high level overview of the cluster. Each card is clicka
 
 ![home-view](./img/01-home-view.png)
 
+## Data loader
+
+The data loader view allows you to load data by building an ingestion spec with a step-by-step wizard. 
+
+![data-loader-1](./img/02-data-loader-1.png)
+
+After picking the source of your data just follow the series for steps that will show you incremental previews of the data as it will be ingested.
+After filling in the requried details on every step you can navigate to the next step by clicking the `Next` button.
+You can also freely navigate between the steps from the top navigation.
```

Comment:
I'd add a comment that some steps will only populate if you click on the "Next button" from the previous step

Reply:
Done